src/aircrack-util/crypto_engine_loader.c

```diff
@@ -198,7 +198,7 @@ void (*dso_ac_crypto_engine_calc_mic)(ac_crypto_engine_t * engine,
 	= NULL;
 #endif
 
-#if defined(CYGWIN)
+#if defined(__CYGWIN__)
```

Comment:
This macro is defined via the Autoconf scripts.

Reply:
Yeah it was an attempt to fix the buildbots. Apparently the reason they fail is that they pass -Werror, which is fairly amusing.

Speaking of which, the cygwin defines are all over the place.

Reply:
Glad you like that. :)